openshift/changeset.go

```diff
@@ -75,7 +76,15 @@ func NewChangeset(platformBasedList, templateBasedList *ResourceList, upsertOnly
 			templateItem.Name,
 		)
 		if err == nil {
-			changes, err := templateItem.ChangesFrom(platformItem)
+			externallyModifiedPaths := []string{}
+			for _, path := range ignoredPaths {
+				pathParts := strings.Split(path, ":")
+				if templateItem.Kind == KindMapping[strings.ToLower(pathParts[0])] {
```

Comment:
Handle case when no colon is there (either by upfront validation in options, or by applying it for all kinds here). Potentially we should allow two colons (to also specify resource name).